CONTRIBUTING.rst

```diff
@@ -503,16 +500,16 @@ should not wait until a minor release. The steps for a patch release
 1. Push the necessary bugfix(es) to the applicable release branch. This
    will generally be the latest release branch (e.g. ``release/0.25``).
 
-2. Update ``__version__.py`` with the next patch increment (e.g.
```

Comment:
I think we'd still need to update `pyvista/_version.py`, right?

Reply:
Yes. We check and review what the next version number is in the diff of `__version__.py`. This automation creates tags and releases from release pull requests.